### DISCUSSION TITLE
I have a problem with tasks 2 and 3 submissions

### DISCUSSION THREAD

**Discussion by angegonzalez** [rramosp/ai4eng.v1#52] [03 - GENERACION DE MODELOS]
I follow all the instructions for submit my code to the grader, but I get this error when I try to run the cell  (happen with tasks 2 and 3):

![image](https://user-images.githubusercontent.com/52757506/125707111-d2a82e57-54c5-4378-9c91-9c943e2229a5.png)

**Comment by Felipe-RA** [comment]
Hello angegonzalez

Please let me see your code, it would  be ideal if you share your notebook

**Reply by angegonzalez** [reply]
I found the solution for task 2:
When I try to use itertools, I wrote this:
`import itertools as it`
And that was the `NameError` problem.

Now, I have a problem with submsssion and that is not clear what I have to submit in task 2, the grader shows me that I have to submit some arrays, but I don't know which ones. 

Also, I have problems with task 3 with the function `predict`

I share my Colab Notebook then:
https://drive.google.com/file/d/1OTjrjpvsiAJugFoLCI0kPP1VIcSN1uBT/view?usp=sharing

**Reply by Felipe-RA** [reply]
Hello

I see in your Notebook that you already solved your problems and found out what needed to be submitted in task 2.
